src/ophys_etl/workflows/db/schemas.py

```diff
@@ -124,6 +124,7 @@ class OphysROI(SQLModel, table=True):
     width: int
     height: int
     is_in_motion_border: bool  # Set at by segmentation
+    has_empty_neuropil_mask: bool = None  # Set at trace extraction
```

Comment:
You might want to make the type `Optional[bool]` though I'm not sure if it's needed.